templates/repo/issue/comment_tab.tmpl

```diff
@@ -28,7 +12,6 @@
 			{{.locale.Tr "loading"}}
 		</div>
 	</div>
-{{end}}
 {{if .IsAttachmentEnabled}}
```

Comment:
Alternatively, we could also extract the lines below into its own subtemplate, so that we don't need to declare them twice.

Reply:
My plan is introducing a "ComboEditor" for Gitea in the future, a ComboEditor could have: markdown toolbar, preview, textarea, dropzone(uploader).

So leave it to future.